source/main.c

```diff
@@ -450,6 +454,9 @@ int do_nand_backup()
 	uint64_t start = gettime();
 	uint64_t lastupdate = 0;
 	puts("Press HOME/START/EJECT to stop.");
+	printf("\nIf the progress bar is stuck for more than 10 seconds, restart your console \n and try again.");
```

Comment:
I would prefer to hear reports about the freeze
Also, half a gigabyte of space will go missing if the app does not get to close the file